service/handlers.go

```diff
@@ -38,12 +38,12 @@ func handleQuery(c *Core, w *ResponseWriter, r *Request) {
 	// The client must look at the return code and interpret the result
 	// accordingly and when it sees a ZNG error after underway,
 	// the error should be relay that to the caller/user.
-	query, err := compiler.ParseOp(req.Query)
+	query, err := compiler.Parse(req.Query)
```

Comment:
```suggestion
	query, err := c.compiler.Parse(req.Query)
```

Reply:
Hmm, I don't know why this passed our tests because c.compiler has the lake pointer but compiler.Parse() doesn't have a lake.

Reply:
Oh, figured it out.  Parse() just builds the AST and the lake work is done when building the query from the AST.